jenkins/templates/route.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.route }}
+---
+apiVersion: v1
+kind: Route
+metadata:
+  annotations:
+    haproxy.router.openshift.io/timeout: 4m
+    template.openshift.io/expose-uri: http://{.spec.host}{.spec.path}
+  name: {{ .Values.appName }}
+spec:
+  tls:
+    insecureEdgeTerminationPolicy: Redirect
+    termination: edge
+  to:
+    kind: Service
+    name: {{ .Values.appName }}
+{{ end }}
```

Comment:
@tylerauerbeck does this need a....
```
status:
  ingress: []
```
?

Reply:
Yeah, just add this to the bottom of the route and you'll be good to go. Otherwise helm will get angry when you try to apply it.